### PR TITLE
Welling/soft assay types start

### PR DESCRIPTION
**Pull request by jpuerto-psc**

### CODE REVIEW

File: src/routes/assayclassifier/assay_types.yaml
Comment (yuanzhou) [inline src/routes/assayclassifier/assay_types.yaml:1]: @jpuerto-psc @jswelling @gesinaphillips, we've deprecated the yaml: https://github.com/hubmapconsortium/search-api/blob/main/src/search-schema/data/definitions/enums/assay_types.yaml Why do we add the same yml in this PR?

Comment (jpuerto-psc) [reply src/routes/assayclassifier/assay_types.yaml:1]: @yuanzhou - this yaml file is used to generate the historical rules. We still need a reference to the historical content in order to make sure that historical datasets are being classified appropriately. This file is **not** used for the new version of metadata schemas.

**Comment by yuanzhou** [comment]
@jpuerto-psc can you update the commit reference to the latest IVT again? The current one `18653712c50317641c5231e6b8827fc4848e0f78` is Joel's branch has a failed CI check. IVT also has 3 new commits.

**Comment by jpuerto-psc** [comment]
@yuanzhou - I have updated this.

**Review by yuanzhou — APPROVED** [review]

**Pull request merged**